turbo.json

```diff
@@ -0,0 +1,16 @@
+{
+    "$schema": "https://turborepo.com/schema.json",
+    "tasks": {
+        "build": {
+            "dependsOn": ["^build"],
+            "outputs": [".next/**", "!.next/cache/**"]
+        },
```

Comment:
We can add these when we have Next.js app setup, but for the moment, let's drop them.

Reply:
Nevermind, ignore this, it is added on the next PR 👍